airflow/example_dags/tutorial_taskflow_api.py

```diff
@@ -45,8 +43,8 @@ def tutorial_taskflow_api():
     located
     [here](https://airflow.apache.org/docs/apache-airflow/stable/tutorial_taskflow_api.html)
     """
-    # [END instantiate_dag]
 
+    # [END instantiate_dag]
```

Comment:
Is this accidental?

Reply:
No, removed blank lines between 2 tags, purely cosmetical change can revert.

Reply:
But you also added a blank line before the END tag.

Reply:
Yes, added it to maintain a gap between the comment and the function. can revert it.